uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/request-modal-details-toil.controller.spec.js

```diff
@@ -102,9 +102,13 @@ define([
         expect(retreiveCachedTOILOptions).not.toBe(false);
       });
 
-      it('sorts TOIL accrual options by value', function () {
-        expect(_.pluck(controller.toilAmounts, 'value')).toEqual(
-          _.pluck(controller.toilAmounts, 'value').sort());
+      it('sorts TOIL accrual options by weight', function () {
+        expect(_.pluck(controller.toilAmounts, function (amount) {
+          return +amount.weight;
+        })).toEqual(
+          _.pluck(controller.toilAmounts, function (amount) {
+            return +amount.weight;
+          }).sort());
```

Comment:
can we remove all this code from the it/expect blocks?

```js

describe('when sorting TOIL accrual options', function () {
  var toilAmounts, toilAmountsSortedByWeight;

  beforeEach(function () {
    toilAmountsSortedByWeight = _.pluck(controller.toilAmounts, function (amount) {
      return +amount.weight;
    }).sort();
    toilAmounts = .pluck(controller.toilAmounts, function (amount) {
      return +amount.weight;
    });
  });

  it('sorts them by weight', function () {
    expect(toilAmounts).toEqual(toilAmountsSortedByWeight);
  });
});
```

Reply:
Done.